docs/make.bat

```diff
@@ -52,6 +53,17 @@ if "%1" == "html" (
 	goto end
 )
 
+if "%1" == "htmlview" (
+    cmd /C %this% html
```

Comment:
If I try and run this with [`cd docs`, `./make.bat htmlview`](https://github.com/radarhere/Pillow/commit/becf01a29b7083d1258f211df93fd2df2df7f103#diff-9e2bbc2329bfb445d0985173b98e981dd7fbf69428e7addd83d453e8d5f3ff2dR192), [I get](https://github.com/radarhere/Pillow/actions/runs/4474922105/jobs/7863853760#step:30:13)
> 'html' is not recognized as an internal or external command,
operable program or batch file.

Reply:
Thanks, I don't have Windows so hadn't tested that. Merged!